esp/src/CMakeLists.txt

```diff
@@ -209,7 +218,6 @@ set ( SRCS
     ${CMAKE_CURRENT_BINARY_DIR}/eclwatch/TargetSelectWidget.js
     ${CMAKE_CURRENT_BINARY_DIR}/eclwatch/TimingPageWidget.js
     ${CMAKE_CURRENT_BINARY_DIR}/eclwatch/TimingTreeMapWidget.js
-    ${CMAKE_CURRENT_BINARY_DIR}/eclwatch/TopologyDetailsWidget.js
```

Comment:
This dojo component was being removed altogether, not just moved elsewhere in the list